docs/pages/07_service.adoc

```diff
@@ -168,25 +168,83 @@ transfer PersonTransfer {
 
 The second example defines the AddressTransfer transfer object with its fields, and each PersonTransfer instance must have an AddressTransfer.
 
-// read-only fields?
-
-=== Default value
-
 Optionally, a <default> value can be specified as an expression for transfer object fields. The <default> value expression is evaluated when a new instance of the transfer object is created, and the field is set according to the default value expression.
 
 *Example:*
 
 ----
 transfer PersonTransfer {
-    field required String firstName ;
-    field required String lastName ;
+    field required String firstName = "John";
+    field required String lastName = "Doe";
     field String midName = "" ;
 }
 ----
 
-The example above complements the previous PersonTransfer example by adding a default value to the "midName" field. The default value of the `midName` is an empty string.
+The example above complements the previous PersonTransfer example by adding a default value to the `firstName`, `lastName` and `midName` fields.
+
+// read-only fields?
+
+=== Mapping configuration
+
+Mapping configuration is a declarative description of how the fields of a transfer object read and/or write specific fields of the entity mapped by the transfer object. Mapping configuration can only be used for mapped transfer objects.
+
+Use the `reads` or `maps` keywords in the field definitions to define the mapping configuration.
+
+*Syntax:*
+
+----
+field [required] <primitive> <name> [= <default>] [(reads|maps) <expression>];
+----
+
+where <primitive> is the name of a domain model primitive, or
+
+----
+field [required] <transfer>[[]] <name> [= <default>]  [(reads|maps) <expression>];
```

Comment:
Is that a valid situation if someone use the dafault value clause and a read or a maps clause? If it is valid, what does it mean?